desk/app/channels.hoon

```diff
@@ -995,6 +1053,13 @@
     =^  caz=(list card)  subs
       (~(handle-wakeup s [subs bowl]) pole)
     (emil caz)
+  ::
+      [%numbers kind=?(%chat %diary %heap) ship=@ name=@ ~]
+    =/  host=ship  (slav %p ship.pole)
+    =/  =nest:c    [kind.pole host name.pole]
+    %-  emit
+    =/  =cage  [%noun !>([%send-sequence-numbers nest])]
+    [%pass pole %agent [host %channels-server] %poke cage]
```

Comment:
Reusing poles (no pun intended) to execute a sequence while avoiding possible wiring mistakes is really nice. Adding to my hoon toolbox.